install.sh

```diff
@@ -16,7 +16,14 @@ set_variable_from_command() {
     DEFAULT_COMMAND=$2
     if [ -z "${!VAR_NAME}" ]
     then
-        result=$(bash -c "${!DEFAULT_COMMAND}") || return $?
+        set +e
```

Comment:
A possible alternative approach (if you want to avoid set +/-e) may be:

```
result=$(bash -c "${!DEFAULT_COMMAND}") || {
    ret=$?; echo "..."; return $ret; 
}
```

Reply:
nice!